docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -17,52 +17,39 @@
     "tags": []
```

Comment:
<blockquote>as it is compute intensive to run experiments on the entire stack due to its size.</blockquote><p> </p><p>Aren't we filtering spatially because of a user request, rather than a memory problem ? AFAIK it runs fine on the entire stack.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Noted and updated accordingly.</p>